prac_05/wimbledon.py

```diff
@@ -0,0 +1,53 @@
+"""
+Emma Naumann
+CP1404 Practical 5
+Estimate: 30 minutes
+Actual:
+"""
+
+FILENAME = "wimbledon.csv"
+INDEX_COUNTRY = 1
```

Comment:
Index of country? You could name it as COUNTRY_INDEX to say that it is the index in which the country is located. Might be something to think about :)

Reply:
Fixed. Thanks :)